notebooks/tel_and_site/subsys_req_ver/tma/SITCOMTN-067_Vel_Accel_Jerk_15Nov23.ipynb

```diff
@@ -0,0 +1,413 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #129.            plt.savefig(str(data_dir / f"MT_Mount_Accel_Jerk_Fail_{timestamp}.pdf"))
</pre><p>This currently saves the plot twice if the slew fails, once as a normal plot and also as a flagged plot. Is this desired behavior? If not, line 114 can be included with an if/else statement</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I removed the duplicate plots and just put in a print statement flagging the out of spec slews.  This had been in there for debug purposes.